pkgs/applications/networking/instant-messengers/jami/client-qt.nix

```diff
@@ -21,7 +21,7 @@
 , qtsvg
 , qtwebengine
 , qtwebchannel
-, withWebengine ? false
+, withWebengine ? true
```

Comment:
Is this now always required? Then we can remove the option completely.

Reply:
I added this option and set it to false because qtwebengine failed to built in hydra and I don't want to force every user to rebuild it. Since qtwebengine builds in hydra now we can enable it by default.